.github/workflows/ci-unix-shared-installed.yml

```diff
@@ -84,6 +87,8 @@ jobs:
 
         cmake . -DCMAKE_PREFIX_PATH=../install
     - name: Prepare libavif with [[nodiscard]] (cmake)
+      # CMake 3.21 is needed to force C23 for [[nodiscard]].
+      if: runner.oldest-cmake == 'false'
```

Comment:
Note: This is a comment, not a request for change.

Here is an example where a `cmake-version` option would be better. We may be able to check if `cmake-version` is >= 3.21. The current check of `runner.oldest-cmake == 'false'` will become incorrect if our minimum required version is bumped to 3.21.

Reply:
Indeed, I just removed one CMake version in `.github/actions/setup-common/action.yml`. Imposing a version in the condition means we need to install two CMake for the same job and dynamically choose one or the other. Or we split the job. I believe I  chose the least confusing solution that is easy for future maintenance: when we bump the minimum CMake version, we might delete oldest-cmake altogether (but as you mention, we will indeed to check all usages of `oldest-cmake` in the scripts).